audio_separator/separator/separator.py

```
@@ -753,30 +753,77 @@ def load_model(self, model_filename="model_mel_band_roformer_ep_3005_sdr_11.4360
+        # If audio_file_path is a string, convert it to a list for uniform processing
+        if isinstance(audio_file_path, str):
+            audio_file_path = [audio_file_path]
+
+        # Initialize a list to store paths of all output files
+        output_files = []
+
+        # Process each path in the list
+        for path in audio_file_path:
+            if os.path.isdir(path):
+                # If the path is a directory, recursively search for all audio files
+                for root, dirs, files in os.walk(path):
```

Ruff (0.8.2)

Loop control variable `dirs` not used within loop body

Rename unused `dirs` to `_dirs`

(B007)